index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mobile-first_Portfolio</title>
+</head>
+
+<body>
+    <header class="header padding-left-right">
+        <a href="#">
+            <h2 class="logo">Thato Mongwe</h2>
+        </a>
+        <nav class="main-nav">
+            <img class="menu-icon" src="img/menu.svg" alt="Hamburger menu icon" />
+        </nav>
+    </header>
+
+    <section class="headline-section padding-left-right">
+        <div class="headline-box">
+            <div class="headline-text-box">
+                <h1 id="headline-primary">
+                    Hey There. <br /> I'm Thato
+                </h1>
+                <p class="headline-title">I am a software developer</p>
+                <p class="headline-description">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <div class="social-icons">
+                    <ul class="social-icons-lists">
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-1.svg" alt="github social-icon" />
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-2.svg" alt="linkedin social-icon" />
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-3.svg" alt="pinterest social-icon" />
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-4.svg" alt="twitter social-icon" />
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-5.svg" alt="mercedes social-icon" />
+                            </a>
+                        </li>
+                    </ul>
```

Comment:
- [ ] Once again please make sure you're following the given [Figma design🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=ovtzuKZtL2D5v5H4-0) for laying out the elements as this will ensure you're future clients you can follow and implement a given design to near perfection.
- Figma design:
  ![image](https://user-images.githubusercontent.com/104968236/234886474-f9fb179e-fd83-47ee-ba5c-a76bb78fef98.png)
- Your design:
  ![firefox_oKBBkRBeEI](https://user-images.githubusercontent.com/104968236/234885710-b1f977a9-95fc-47b5-b17c-7ac1777e1ed2.png)
- As you can see the icons should be aligned horizontally and make sure to remove the bullet points as well. 👍